rclcpp/test/rclcpp/test_loaned_message.cpp

```diff
@@ -77,7 +77,7 @@ TEST_F(TestLoanedMessage, release) {
     // of the data after a call to release.
   }
 
-  ASSERT_EQ(42.0f, msg->float64_value);
+  ASSERT_EQ(42.0f, msg.get()->float64_value);
```

Comment:
Is this needed? Does `unique_ptr` not implement `operator->`?

https://en.cppreference.com/w/cpp/memory/unique_ptr/operator*

Reply:
Thank you.
> Is this needed?

No. Updated.